### PR TITLE
boil down routes

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

**Comment by ScottDowne** [comment]
r?

**Comment by ScottDowne** [comment]
@jbuck After this lands, I figure if you do one error handle, I'll copy that pattern and do the rest.

File: pages/sequential.jsx [hunk 1]
```diff
@@ -537,8 +537,8 @@ var Sequential = React.createClass({
                     </div>
                   </div>
                   <div className="row" id="payment-type-row">
-                    <PaypalButton />
```
Comment (jbuck) [inline pages/sequential.jsx]: Should this change be in this patch?

File: routes/index.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
+var httpRequest = require('request');
+var stripe = require("./stripe");
+var paypal = require("./paypal");
+
+var routes = {
+  "signup": function(request, reply) {
+    var transaction = request.payload || {};
+    httpRequest({
+      url: 'http://donate-mozilla.herokuapp.com/thank-you/',
```
Comment (jbuck) [inline routes/index.js]: The host is hard-coded and I think the path is wrong here too

Comment (alicoding) [inline routes/index.js]: https://github.com/mozilla/donate.mozilla.org/issues/138

**Comment by jbuck** [comment]
@ScottDowne r+

**Pull request merged**